web/src/package.json

```diff
@@ -55,7 +56,9 @@
     "react-transition-group": "4.4.2",
     "recharts": "2.1.8",
     "redux-thunk": "2.4.0",
-    "remark-gfm": "3.0.0"
+    "remark-gfm": "3.0.0",
+    "urql": "2.2.0",
+    "wonka": "4.0.15"
```

Comment:
what's this?

Reply:
https://wonka.kitten.sh/

Used by the exchange code:
https://formidable.com/open-source/urql/docs/advanced/authoring-exchanges/

Reply:
This might be a better one for how urql uses it:
https://formidable.com/open-source/urql/docs/architecture/#the-wonka-library